src/learn_to_pick/pick_best.py

```diff
@@ -340,26 +340,37 @@ def run(self, **kwargs) -> Dict[str, Any]:
     def create(
         cls: Type[PickBest],
         policy: Optional[base.Policy] = None,
-        llm = None,
+        llm=None,
         selection_scorer: Union[base.AutoSelectionScorer, object] = SENTINEL,
-        metrics_step: int = -1,
-        metrics_window_size: int = -1,
-        **policy_args: Any,
+        **kwargs: Any,
     ) -> PickBest:
         if selection_scorer is SENTINEL and llm is None:
             raise ValueError("Either llm or selection_scorer must be provided")
         elif selection_scorer is SENTINEL:
             selection_scorer = base.AutoSelectionScorer(llm=llm)
-        if policy and any(policy_args):
+
+        policy_args = {
+            "feature_embedder": kwargs.pop("feature_embedder", None),
+            "vw_cmd": kwargs.pop("vw_cmd", None),
+            "model_save_dir": kwargs.pop("model_save_dir", None),
+            "reset_model": kwargs.pop("reset_model", None),
+            "vw_logs": kwargs.pop("vw_logs", None),
+        }
+
+        if policy and any(policy_args.values()):
             logger.warning(
-                f"{list(policy_args.keys())} will be ignored since nontrivial policy is provided"
+                f"{[k for k, v in policy_args.items() if v]} will be ignored since nontrivial policy is provided, please set those arguments in the policy directly if needed"
             )
+        
+        if policy_args["model_save_dir"] is None:
+            policy_args["model_save_dir"] = "./"
+        if policy_args["reset_model"] is None:
+            policy_args["reset_model"] = False
 
         return PickBest(
-            policy = policy or PickBest.create_policy(**policy_args),
-            selection_scorer = selection_scorer,
-            metrics_step = metrics_step,
-            metrics_window_size = metrics_window_size,
+            policy=policy or PickBest.create_policy(**policy_args, **kwargs),
```

Comment:
it seems wrong that we are providing kwargs to both policy and RlLoop constructor - hard to come up with real usage when it may work